x/programs/rust/wasmlanche_sdk/src/host/mod.rs

```diff
@@ -39,7 +39,7 @@ extern "C" {
 /// `value_ptr` + `value_len` point to valid memory locations.
 #[must_use]
 pub unsafe fn store_bytes(
-    ctx: Context,
+    ctx: State,
```

Comment:
```suggestion
    state: State,
```

Reply:
There are other places for this too. Should be easy to do a global search and replace